src/Components/Analyzers/test/ComponentsTestDeclarations.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Components.Analyzers
+{
+    public static class ComponentsTestDeclarations
+    {
+        public static readonly string Source = $@"
+    namespace {typeof(ParameterAttribute).Namespace}
```

Comment:
This is a looming maintenance problem, but I chose not to solve it in this PR. In our other analyzers we use a more mature test infrastructure, and we use the actual product code for testing.